cpp/arcticdb/python/python_module.cpp

```diff
@@ -236,6 +236,9 @@ void register_error_code_ecosystem(py::module& m, py::exception<arcticdb::Arctic
             internal_exception(msg.c_str());
         }
     });
+    // DuplicateKeyException and PermissionException are subclasses of StorageException so keep the hierarchy in Python too
```

Comment:
drop the comment?

Reply:
Done in [59436ce](https://github.com/man-group/ArcticDB/pull/1411/commits/59436ceca3bb044cf4e6412c2eceaebf9080880f)